lib/CompositeController.js

```diff
@@ -86,11 +108,37 @@ module.exports = class CompositeController extends BaseController {
       }
     }
 
-    let krm = await this.kubeClass.getKubeResourceMeta(child.apiVersion, child.kind, 'update');
+    if (!childName) {
+      return {
+        statusCode: 400,
+        body: {
+          kind: 'Status',
+          apiVersion: 'v1',
+          metadata: {},
+          status: 'Failure',
+          message: `Invalid kubernetes resource, 'metadata.name: ${childName}' must not be empty`,
```

Comment:
Won't ${childName} be undefined in this message?

Reply:
yes. that was kind of the point. but for debuging future problems, i like to print it out just in case